src/doubleTrouble.js

```diff
@@ -26,7 +26,13 @@
  * @return {number[]} - array of remained numbers
  */
 function doubleTrouble(array, target) {
-  // write code here
+  for (let i = 0; i <= array.length; i++) {
+    if (array[i] + array[ i + 1 ] === target) {
```

Comment:
[do not add spaces inside square brackets](https://mate-academy.github.io/style-guides/javascript.html#whitespace--in-brackets)